circle.yml

```diff
@@ -1,6 +1,14 @@
 machine:
   services:
     - docker
+  environment:
+    GODIST: "go1.8.3.linux-amd64.tar.gz"
```

Comment:
Can i ask what is the reason to use this version instead of latest?

Reply:
Yes, that is cherry-picked from the master and it's a working version :)

Reply:
We use the same version on master. With the current setup without a version the binary does not work on macos. I'm not sure what version is used then.